### ISSUE TITLE
Host always shown Dead after first Discussion round

### ISSUE THREAD

**Issue by xAffectum** [automuteus/amonguscapture#155]
**Describe the bug**
After the first discussion round when going back to tasks, the host always shown as dead in capture software even if he is or not.
Is this a normal behaviour?
Because of this the Host always staying Server Muted in the next discussion rounds.

**Expected behavior**
Only Players who are killed should be shown as dead.

**Comment by xAffectum** [comment]
Among Us via Epic Games

**Comment by CarbonNeuron** [comment]
Is this still a problem in the latest update? Epic games is now a 32bit executable.

**Comment by ewigkeit1204** [comment]
I encountered this issue using 3.2.1

**Comment by ewigkeit1204** [comment]
Log is following when this issue occured.

```
12:16:34.4483|INFO|AUCapture_WPF.MainWindow|State change: {"NewState":"DISCUSSION"}
12:18:54.5578|DEBUG|AUCapture_WPF.MainWindow|{"Action":"Exiled", "Name":"<HostUser>", "IsDead":false, "Disconnected":false, "Color":"Cyan"}
12:18:54.5578|INFO|AUCapture_WPF.MainWindow|State change: {"NewState":"TASKS"}
```

When discussion is ended, host user is judged as exiled.

**Comment by ewigkeit1204** [comment]
This issue not affects Steam version (current v2021.3.5s). It works correctly.
I confirmed this issue affects Microsoft Store version (current v2021.2.21m).

**Comment by a01sa01to** [comment]
I also encountered this bug, using Among Us via Microsoft Store (v2021.2.21m) and AUCapture (3.2.1).

Log: 
```
13:11:08.7333|INFO|AUCapture_WPF.MainWindow|State change: {"NewState":"DISCUSSION"}  // First Discussion
13:11:21.0366|DEBUG|AUCapture_WPF.MainWindow|{"Action":"Exiled", "Name":"<Host>", "IsDead":false, "Disconnected":false, "Color":"<Color>"}  // In fact the Host was not exiled
13:11:21.0366|INFO|AUCapture_WPF.MainWindow|State change: {"NewState":"TASKS"}
13:11:34.8429|INFO|AUCapture_WPF.MainWindow|State change: {"NewState":"DISCUSSION"}  // Second Discussion
13:11:45.3856|DEBUG|AUCapture_WPF.MainWindow|{"Action":"Exiled", "Name":"<Host>", "IsDead":false, "Disconnected":false, "Color":"<Color>"}  // Host was really exiled
13:11:45.3856|INFO|AUCapture_WPF.MainWindow|State change: {"NewState":"TASKS"}
```

I can't check the behavior with Among Us on other platforms, sorry.

The log is the same that `{"Action":"Exiled", "Name":"<Host>", "IsDead":false, "Disconnected":false, "Color":"<Color>"}`, so I think we need another way to detect players' status in order to solve this issue.

**Comment by a01sa01to** [comment]
Hello.
I noticed that in my environment, this bug has been fixed with Among Us via Microsoft Store (v2021.4.14m) and AUCapture (4.0.2).
If anyone else doesn't encounter this bug, I think this issue can be closed.
Thanks.
